servers/src/main/java/tachyon/worker/block/BlockWorker.java

```diff
@@ -190,10 +204,26 @@ private TThreadPoolServer createThriftServer() {
   }
 
   /**
+   * Get the worker start time (in UTC) in milliseconds.
+   * @return the worker start time in milliseconds
+   */
+  public long getStartTimeMs() {
+    return mStartTimeMs;
+  }
+
+  /**
+   * Gets the meta data of the entire store.
+   * @return store meta data
+   */
+  public BlockStoreMeta getStoreMeta() {
+    return mBlockDataManager.getStoreMeta();
+  }
+
+  /**
    * Helper method to get the {@link java.net.InetSocketAddress} of the worker.
    * @return the worker's address
    */
-  private InetSocketAddress getWorkerAddress() {
+  public InetSocketAddress getWorkerAddress() {
```

Comment:
Instead of making this public, could another method be added that gets the worker address from the NetAddress (`mWorkerNetAddress`)?